### ISSUE TITLE
AMP Compatible.

### ISSUE THREAD

**Issue by fkraefft** [tailwindcss/tailwindcss#486]
The '!important' tag in the outline of the preflight.css file it's not compatible with the new [ampproject](https://www.ampproject.org/) to make websites faster. It's necessary to have it?

```
/**
 * Suppress the focus outline on elements that cannot be accessed via keyboard.
 * This prevents an unwanted focus outline from appearing around elements that
 * might still respond to pointer events.
 */

[tabindex="-1"]:focus {
  outline: none !important;
}
```

Btw, nice job.

**Comment by hacknug** [comment]
That comes from `suitcss`'s base project si I guess it'd be better to ask there: https://github.com/suitcss/base/blob/master/lib/base.css

**Comment by fkraefft** [comment]
I've made a issue there.
